### DISCUSSION TITLE
Будет ли полезен roadmap в проекте?

### DISCUSSION THREAD

**Discussion by ivanstrilets** [doka-guide/content#2098] [Идеи] [answered]
Представляется мне, что было бы здорово иметь roadmap. В котором человек мог бы идти последовательно изучая материал. Конечно если такую цель преследует doka. А так же, можно было бы отмечать прочитанные статьи в этом самом roadmap.

**Comment by solarrust** [comment]
Предлагаю перевести этот вопрос в обсуждения и пригласить всех заинтересованных к обсуждению. Что думаете?

**Comment by ivanstrilets** [comment]
@solarrust думаю стоит так сделать

**Comment by pepelsbey** [comment]
И вот теперь это дискуссия :)

**Comment by solarrust** [comment, marked as answer]
Выскажу своё мнение по поводу дорожной карты.
Идея, как мне кажется, очень хорошая. Но есть вопрос «А судьи кто?». Составить хороший роадмап для всех-всех очень сложно. Каждый обучается тому, что ему интересно. А если мы закрепим важные на наш взгляд темы в роадмапе, то можем направить человека не по тому пути. 
Мы постарались структурировать информацию в Доке и разделили каждый раздел на подразделы, выстроили их в порядке востребованности. Если человек просто хочет читать всё подряд, то может ориентироваться на эти подразделы. Но такой сценарий использования справочника крайне редкий. Обычно к нам заходят найти конкретную информацию. 
Возможно, позже мы вернёмся к этому вопросу, но пока одной из главных наших целей является максимальное покрытие веб-платформы статьями. 

Спасибо большое что подняли эту тему 🙏
Мы увидели важность этой темы для комьюнити!

**Reply by ivanstrilets** [reply]
Быть может роадмап на разделы и не требуется. А вот по технологиям очень даже может и пригодится. Но, действительно, у меня нет данных с какими потребностями приходят в doka. И если посетитель нуждается только в конкретном разделе, смысла это не имеет.
